README.md

```diff
@@ -450,24 +439,25 @@ If you intend to install the clidriver manually, Following are the details of th
 
 #### <a name="LicenseDetails"></a> CLIDriver and Client license versions for Specific Platform and Architecture
 
-|Platform      |Architecture    |Cli Driver               |Supported     |Version      |
+|Platform      |Architecture    |Cli Driver               |Supported     |Version        |
 | :---:        |  :---:         |  :---:                  |  :---:       | :--:
-|AIX           |  ppc           |aix32_odbc_cli.tar.gz    |  Yes         | V11.5.8     |
```

Comment:
@Earammak Change all v11.5.8 to v11.5.9 as v11.5.9 is the default driver for these platforms. thanks.